check_patroni/types.py

```diff
@@ -60,14 +60,16 @@ def rest_api(self: "PatroniResource", service: str) -> Any:
                 _log.debug(e)
                 continue
             # The status code is already displayed by urllib3
-            _log.debug("api call data: %(data)s", {"data": r.text})
+            _log.debug(
+                "api call data: %(data)s", {"data": r.text if r.text else "<Empty>"}
+            )
 
             if r.status_code != 200:
```

Comment:
Also, here:
```python
if not r.ok:
```
would be more appropriate, as any 20x status code is not an error.

Reply:
patroni's [api documentation](https://patroni.readthedocs.io/en/latest/rest_api.html) mentions than
a 200 return codeis expected for most api calls.